bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: Expose instance type to the data plane

## Description
The data plane needs to know the type of an instance (instance_type).
[MGDSTRM-7957](https://issues.redhat.com/browse/MGDSTRM-7957)
## Verification Steps
1. Start a cluster (ocm or crc).
2. Deploy kas-fleet-manager to the cluster. 
3. Observe the cluster YAML for the newly added labels.

## Checklist (Definition of Done)
